files/sdw-admin.py

```diff
@@ -70,7 +69,7 @@ def install_pvh_support():
     TODO: install this via package requirements instead if possible
     """
     try:
-        subprocess.run(["sudo", "qubes-dom0-update", "-y", "-q", "grub2-xen-pvh"])
+        subprocess.run(["sudo", "qubes-dom0-update", "-y", "-q", "grub2-xen-pvh"], check=False)
```

Comment:
wouldn't we want `check=True` here in order to trigger the exception below if the update is unsuccessful?